LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 AGRC
+Copyright (c) 2020 UGRC
```

Comment:
**suggestion**: I think you can drop the year to remove the need to update that ever year and keep the same protections.
```suggestion
Copyright (c) UGRC
```